src/lib/verifier/DAVerifier.sol

```diff
@@ -83,20 +76,23 @@ library DAVerifier {
     /// @param _sharesProof The proof of the shares to the data root tuple root.
     /// @param _root The data root of the block that contains the shares.
     /// @return `true` if the proof is valid, `false` otherwise.
+    /// @return an error code if the proof is invalid, 0 otherwise.
     function verifySharesToDataRootTupleRoot(IDAOracle _bridge, SharesProof memory _sharesProof, bytes32 _root)
         external
         view
-        returns (bool)
+        returns (bool, uint8)
```

Comment:
Instead, return a `ErrorCodes` and make a new error code for "no error"?

Reply:
[7096a57](https://github.com/celestiaorg/quantum-gravity-bridge/pull/189/commits/7096a577d015a98a89d05c9bc3dc681211d89e71)